pulsar-broker/src/test/java/org/apache/pulsar/client/impl/SequenceIdWithErrorTest.java

```diff
@@ -39,7 +39,7 @@
     /**
      * Test that sequence id from a producer is correct when there are send errors
      */
-    @Test(enabled = false)
+    @Test
```

Comment:
did you try with `invocationCount=20`? I think earlier when I spent sometime to debug this issue,  I tried to close-topic on subscription-fenced ledger exception but still test was failing intermittently and I realized topic was stuck at [schemaVersionFuture](https://github.com/apache/incubator-pulsar/blob/master/pulsar-broker/src/main/java/org/apache/pulsar/broker/service/ServerCnx.java#L832) so, produce was timing out.

Reply:
Yes, I tried with 300 invocation. This pr is just part of the fix, the actual fix for schema future is at #2148

Reply:
I see, I didn't see it. make sense.